flask_and_kafka/producer.py

```diff
@@ -14,18 +17,23 @@ def __init__(self, app: Flask = None) -> None:
     def init_app(self, app: Flask) -> None:
         app.extensions["kafka_producer"] = self
         self.producer = Producer(app.config["KAFKA_PRODUCER_CONFIGS"])
-        self.producer_logger = producer_logger(
-            name="producer_logger",
-            file=app.config.get(
-                "KAFKA_PRODUCER_LOG_PATH", "logs/kafka_producer.log"
-            ),
-        )
+        logger_name = 'producer_logger'
```

Comment:
I recommend having this part separated into a dedicated method to make testing and user interaction with it easier. Additionally, it would be excellent if we could have documentation for it 😃.